backend/modules/query_engines/__init__.py

```diff
@@ -0,0 +1,5 @@
+from backend.modules.query_engines.default_engine import DefaultQueryEngine
+
+
+class QueryEngineModule:
+    modules = [DefaultQueryEngine]
```

Comment:
We'll need to figure out some automatic registration because we would like to not touch any of the files here when adding code in clones
Probably some metaclass approach that can store all children in base classess static attribute, it can also do conflict checking if routes are same and clashing

Reply:
https://github.com/truefoundry/docs-qa-playground/pull/43/files#r1487350113